src/azure/storage/loader.rs

```diff
@@ -19,7 +19,16 @@ impl Loader {
     /// Create a new loader via config.
     pub fn new(config: Config) -> Self {
         Self {
-            config,
+            config: Config::default().from_env(),
```

Comment:
Will change the `new` function after opendal side apply this change, currently we are using this branch for test.